### PR TITLE
Admin Update Material

### PR DESCRIPTION
**Pull request by vhhieu95**
Admin Update Material

### CODE REVIEW

File: app/Http/Controllers/MaterialController.php [hunk 1]
```diff
@@ -40,6 +42,52 @@ public function index()
...
     /**
+     * Show the form for editing the specified resource.
+     *
+     * @param int $id of material
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $material = $this->material->findOrFail($id);
```
Comment (ghost) [inline app/Http/Controllers/MaterialController.php:54]: Pls using Try Catch for call function ```findOrFail```

File: app/Http/Controllers/MaterialController.php [hunk 2]
```diff
@@ -40,6 +42,52 @@ public function index()
...
+     *
+     * @param MaterialUpdateRequest $request update material
+     * @param int                   $id      id of material
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function update(MaterialUpdateRequest $request, $id)
+    {
+        
+        $material = $this->material->findOrFail($id);
```
Comment (ghost) [inline app/Http/Controllers/MaterialController.php]: Pls using Try Catch for call function ```findOrFail```

**Review by ghost — COMMENTED** [review]
Pls check my comment

File: app/Http/Controllers/MaterialController.php (hunk 1 above)
Comment (vhhieu95) [reply app/Http/Controllers/MaterialController.php:54]: I think I don't need try catch for function findOrFail because have exception handel

**Review by ghost — APPROVED** [review]

**Pull request merged**